packages/components/addon/components/hds/dropdown/list-item/checkbox.hbs

```diff
@@ -0,0 +1,8 @@
+<li class="hds-dropdown-list-item hds-dropdown-list-item--checkbox">
+  <Hds::Form::Checkbox::Field @id={{@id}} @name={{@name}} @value={{@value}} ...attributes as |F|>
+    <F.Label>{{yield}}</F.Label>
+  </Hds::Form::Checkbox::Field>
+  {{#if @resultCount}}
+    <Hds::BadgeCount class="hds-dropdown-list-item__count" @size="small" @type="filled" @text={{@resultCount}} />
```

Comment:
This is my fault. I used the BadgeCount component as a starting point in the explorations because we were looking at using them directly and for this variation of the design, I just detached and overrode the values. I'll make sure the built component is just plain text with a container. It makes sense to me that the implemented version does not use the BadgeCount component.

Reply:
thank you both! will update to use a plain element then

Reply:
[We're now using a plain element for the result count](https://github.com/hashicorp/design-system/pull/1185/commits/d3d7cc80906c0163b0d39d66f22f6b0932068704) (replaced `BadgeCount` with `<span>` and applied similar style)